app/Observers/ClientObserver.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace App\Observers;
+
+use App\Models\Client;
+use App\Notifications\ClientCreatedNotification;
+use Auth;
+
+class ClientObserver
+{
+    /**
+     * Handle the Client "created" event.
+     *
+     * @param \App\Models\Client $client
+     * @return void
+     */
+    public function saving(Client $client)
```

Comment:
@faxunil there are separate methods like `creating()` and `updating()` and `deleting()` from what I remember.